app/views/customers/show.html.erb

```diff
@@ -1,5 +1,8 @@
-<p style="color: green"><%= notice %></p>
-# Just for 1 client - order
+<% if flash[:notice].present? %>
+  <div class="alert alert-success" role="alert">
+    <%= flash[:notice] %>
+  </div>
+<% end %>
```

Comment:
Heads up that since you display flash messages in `app/views/layouts/application.html.erb` and this view is rendered within that layout, displaying flash messages here means they are displayed twice on the page. To correct this, you can remove this one and the one in `app/views/customers/index.html.erb`. 

<img width="1101" alt="Screenshot 2024-09-23 at 12 29 24 PM" src="https://github.com/user-attachments/assets/8baa219b-e017-4c94-915e-8edefa8f8686">